vdr package uses some macros like HOST_NAME_MAX, NAME_MAX, which are defined
in limits.h.
Needs to be explicitly included on ppc64le and for all archs for debug build.
----

--- a/config.h
+++ b/config.h
@@ -19,6 +19,7 @@
 #include "i18n.h"
 #include "font.h"
 #include "tools.h"
+#include <limits.h>
 
 // VDR's own version number:
 
--- a/tools.c
+++ b/tools.c
@@ -27,6 +27,7 @@
 #include <utime.h>
 #include "i18n.h"
 #include "thread.h"
+#include <limits.h>
 
 int SysLogLevel = 3;
 
